internal/manifest/crypto_test.go

```diff
@@ -0,0 +1,63 @@
+// Copyright 2024 Edgeless Systems GmbH
+// SPDX-License-Identifier: AGPL-3.0-only
+
+package manifest
+
+import (
+	"crypto/rand"
+	"crypto/rsa"
+	"crypto/sha256"
+	"testing"
+
+	"github.com/edgelesssys/contrast/internal/userapi"
+	"github.com/stretchr/testify/require"
+)
+
+func TestEncryptDecryptSingleKey(t *testing.T) {
```

Comment:
Remember that we use table driven tests by default (but no need to change this now).

Reply:
I tried to use the `testCase` pattern, but eventually realized that we want the full cartesian product. The tests for wrong key and wrong label are somewhat special and don't seem to fit naturally into a test table.